### ISSUE TITLE
I think 12.2 (definition of determinant) omits an important part of the proof

### ISSUE THREAD

**Issue by mrfoogles** [vEnhance/napkin#186]
12.2 defines the determinant by defining a function $\wedge^n(V) \to \wedge^n(V)$ by taking $v_1 \wedge v_2 \wedge \ldots \wedge v_n$ to $T(v_1) \wedge T(v_2) \wedge \ldots \wedge T(v_n)$. This is not too hard to show as linear, but it is actually pretty hard to show that this is even defined as a function, given $(cv) \wedge w = c \wedge (cw)$ and so on. For a general wedge space, T has to be (at least almost) linear for this to map equivalent elements to equivalent elements, and I think a lot of the reason that matrices have determinants at all is hidden in this step of the proof. For it to work, if two parallelograms have equal orientation and area (and are the same in the wedge product space), applying T must map each of them to the same new orientation and area. I didn't really make as much sense to me the first time I read the section, because we're just suddenly on "linear maps are linear in wedge spaces" without much buildup and I couldn't see how we got there.

**Comment by vEnhance** [comment]
This is a reasonable comment, but I'm confused about the line

> T has to be (at least almost) linear for this to map equivalent elements to equivalent elements,

Isn't $T$ given to be a linear map by definition? The construction isn't defined if $T$ isn't linear.

**Comment by mrfoogles** [comment]
Yeah, I'm saying I think the proof should mention that. Right now the proof is "T is linear -(trivially)-> The function ${\bigwedge}^{n}$ is linear -> determinants work like they do." I think it is trivial to prove that ${\bigwedge}^{n}(T)$ is a function, it is linear, but it's nontrivial to prove that ${\bigwedge}^{n}(T)$ is a function in the first place. But then that's the part that the proof skips. For example, to prove that ${\bigwedge}^{n}(T)$ is a function, you have to prove that $T(cv) \wedge T(\frac{1}{c}w) = T(v) \wedge T(w)$. This corresponds to proving that ${\bigwedge}^{n}(T)$ gives the same area for stretched/squished versions of a parallelogram with the same area. But then if you don't do that part, it doesn't really tell you why determinants work like they do.

**Comment by vEnhance** [comment]
Got it. That's fair.

I wrote a proposal for an edit at:

https://github.com/vEnhance/napkin/pull/187/files

Do you think that's satisfactory? Or should I elaborate even more?

**Comment by mrfoogles** [comment]
I think that definitely makes it more clear what's happening. You could probably go into some of why that works (I think the whole stretching-squishing thing is neat) but it depends on your style. I (not knowing linear algebra) feel like why the proof works is it proves that T treats all parallelograms with the same area and orientation the same (maps them to something with the same area and orientation). Then the squishiness part (showing that T preserves the scaling axiom of the wedge product) is pretty simple, but I feel like the other part, proving that $\bigwedge$ is invariant under the addition axioms of wedge product, is harder; I know how to literally prove T preserves the axioms, but I don't really know why that works in the context of actual parallelograms.

**Comment by vEnhance** [comment]
> I know how to literally prove T preserves the axioms, but I don't really know why that works in the context of actual parallelograms.

imo it's the other way around and the parallelograms are just motivation.

This is a common trope in the way math is set up. You have some intuition about an object you wish to define, and then you write down a wishlist of properties that "should" follow from your intuition (like $v \wedge v = 0$). But it turns out the properties are enough to work with (e.g. they imply $v \wedge w = -w \wedge v$), and so for the definition, you just define an abstract object satisfying all the properties on your wishlist. Thereafter the intuition plays no "official" role.

I'll add a remark to this effect. Thanks for the feedback.
